lib/src/effect/default_effector.dart

```diff
@@ -10,7 +24,8 @@ class DefaultEffector implements Effector {
       result = effects.any((eft) => eft == Effect.Allow);
     } else if (expr == '!some(where (p_eft == deny))') {
       result = !effects.any((eft) => eft == Effect.Deny);
-    } else if (expr == 'some(where (p_eft == allow)) && !some(where (p_eft == deny))') {
+    } else if (expr ==
+        'some(where (p_eft == allow)) && !some(where (p_eft == deny))') {
```

Comment:
I'd suggest to use constants(or enums) instead of hardcoded strings. You can refer to the docs for the names of common policy effects. Moreover, you could also refactor this code into a clean switch case paradigm

Reply:
Go code is also like this, so I think it's OK for now. https://github.com/casbin/casbin/blob/master/effector/default_effector.go

Reply:
Alright